scenarios/loop/scenario.py

```diff
@@ -18,7 +18,35 @@
                 ): 1
             },
         )
-        for i in range(10)
+        for i in range(5)
     ]
 )
-gen_scenario(t.Scenario(traffic={"basic": traffic}), output_dir=Path(__file__).parent)
+
+laner_actor = t.SocialAgentActor(
+    name="keep-lane-agent", agent_locator="zoo.policies:keep-lane-agent-v0",
+)
+
+slow_actor = t.SocialAgentActor(
+    name=f"slow-agent",
+    agent_locator="zoo.policies:non-interactive-agent-v0",
+    policy_kwargs={"speed": 10},
+)
+
+gen_scenario(
+    t.Scenario(
+        traffic={"basic": traffic},
+        social_agent_missions={
+            "all": ([laner_actor], [t.Mission(route=t.RandomRoute())])
+        },
+        bubbles=[
+            t.Bubble(
+                zone=t.PositionalZone(pos=(50, 0), size=(10, 15)),
+                margin=5,
+                actor=slow_actor,
+                pinned_actor_id=t.Bubble.to_actor_id(laner_actor, mission_group="all"),
+                pinned_offset=(-7, 10),
```

Comment:
I didn't understand what `pinned` meant the first time around, how about `follow_actor_id` or `tracking_actor_id` or `travel_with_actor_id`.